src/libcmd/installables.hh

```diff
@@ -131,6 +131,7 @@ struct Installable
         OperateOn operateOn,
         const std::vector<std::shared_ptr<Installable>> & installables);
```

Comment:
```suggestion
        const Installables & installables);
```

and there are probably more places that can use the `Installables` type.

Reply:
I started to make this change, but the compiler was not happy with how I was doing it... we can leave this to another PR? or during hacking session?